bigquery_etl/cli/query.py

```diff
@@ -935,7 +935,7 @@ def _run_query(
             logging.error(e)
             sys.exit(1)
         except FileNotFoundError:
-            logging.warning("No metadata.yaml found for {}", query_file)
+            logging.warning("No metadata.yaml found for %s", query_file)
```

Comment:
Any reason not to use an f-string?

Reply:
My understanding that `%` is the preferred approach in logging as it is lazily evaluated.

Reply:
Personally I'd prefer the obviousness of f-strings even at a tiny performance loss, but this color looks fine for a bikeshed as well

Reply:
In this case not really other than it feels like the proper way to do it since logging has this built in.  More generally doing this means the string formatting is only done if it's actually logged based on the logging config which is better for performance.  Not relevant here but kind of interesting to know